recipes/nvitop/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "nvitop" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nvitop-{{ version }}.tar.gz
+  sha256: 3d9be8b355fae4960e9bfcda6111a0cde601880a4deaf1129e95f84a250d9b4b
+
+build:
+  skip: true  # [py<35]
```

This looks like it could be noarch Python:
```suggestion
  noarch: python
```